launchers/connectors/broker-server-dev/build.gradle.kts

```diff
@@ -0,0 +1,38 @@
+plugins {
+    `java-library`
+    id("application")
+    alias(libs.plugins.shadow)
+}
+
+val edcVersion: String by project
+val edcGroup: String by project
+
+dependencies {
+    // Control-Plane
+    implementation("${edcGroup}:control-plane-core:${edcVersion}")
+    implementation("${edcGroup}:data-plane-selector-core:${edcVersion}")
+    implementation("${edcGroup}:api-observability:${edcVersion}")
+    implementation("${edcGroup}:configuration-filesystem:${edcVersion}")
+    implementation("${edcGroup}:control-plane-aggregate-services:${edcVersion}")
+    implementation("${edcGroup}:http:${edcVersion}")
+    implementation("${edcGroup}:dsp:${edcVersion}")
+    implementation("${edcGroup}:json-ld:${edcVersion}")
+
+    // JDK Logger
+    implementation("${edcGroup}:monitor-jdk-logger:${edcVersion}")
```

Comment:
do we re-use the logging style of the connector?

Did you check, with which system / style the broker CE did logging before and compared it to the connector CE?

For example, our EE does JSON logging which would definitely differ

Reply:
The `logging.properties` of the broker and the launchers are identical.

Reply:
And they both used `monitor-jdk-logger`.

The broker dev and server used the same config file:
https://github.com/sovity/edc-broker-server-extension/blob/main/connector/build.gradle.kts

So both with the JDK/monitor lib.

So unless there's some funky config at runtime I think they will log the same way.